tool/tsh/tsh.go

```diff
@@ -3114,19 +3176,40 @@ func serializeProfiles(profile *client.ProfileStatus, profiles []*client.Profile
 	return string(out), nil
 }
 
-func printProfiles(debug bool, profile *client.ProfileStatus, profiles []*client.ProfileStatus) {
+func getTshEnv() map[string]string {
+	env := map[string]string{}
+	envVars := []string{proxyEnvVar, clusterEnvVar, siteEnvVar, kubeClusterEnvVar, teleport.EnvKubeConfig}
```

Comment:
Can we move this into a global level variable that sits next to where these env vars are defined? Otherwise noone will know that they have to update it here as well if for example adding a new one.